SiteLogToGeodesyML/modified-schemas/geodeticEquipment.xsd

```diff
@@ -69,15 +69,19 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
                     <!-- -->
                     <!-- Manufacturer Name -->
                     <!-- -->
+                    <element minOccurs="0" name="manufacturerName" type="string"/>
```

Comment:
Why was there previously no manufacturerName?

Reply:
Not sure why there is no these fields in v0.4. But these fields were in v0.4 in GeodesyML project.

I updated all xsd files by copying the contents from 0.5 version of GeodesyML project.

Reply:
The schema files in this project sit in a directory called modified-schemas, which means that for some reason they were intentionally modified, possibly to get around some limitation of pyxb. I would be reluctant to add them back without understanding why they were removed. I will look at the commit history to try to work why these fields were removed. @dotzhou may be able to offer some advice.

Reply:
Ted explained to me why these fields are removed from the schema: the code using pyxb could not pass the compilation if using standard schemas. So Ted modified some of the schemas to make it work. Otherwise, we only need one version of schemas for this project.

It seems now that the compiling and tests work fine after restoring these fields.

Reply:
Yes, I was wondering what the particular reason was for removing these particular fields. I guess we don't know, since pyxb now works just fine with them back in. Could you please send a separate pull request to add these fields back in on their own, so that we can test that change independently.

Reply:
We need to keep separate the changes we want to make anyway from the changes we want to make as part of migration to geodesyml 0.5.

Reply:
These changes have been removed and I will add these fields back in a new PR.